src/core_modules/capture-core/components/Pages/Search/SearchForm/SearchForm.container.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import { connect } from 'react-redux';
+import { SearchFormComponent } from './SearchForm.component';
+import { searchPageActionTypes } from '../SearchPage.container';
+import { actionCreator } from '../../../../actions/actions.utils';
+import type { DispatchersFromRedux, OwnProps, Props, PropsFromRedux } from './SearchForm.types';
+
+
+const mapStateToProps = (state: ReduxState): PropsFromRedux => {
+    const { forms, searchPage: { searchStatus } } = state;
+    return {
+        forms,
+        searchStatus,
+    };
+};
+
+const mapDispatchToProps = (dispatch: ReduxDispatch): DispatchersFromRedux => {
+    const {
+        VIA_UNIQUE_ID_ON_SCOPE_TRACKED_ENTITY_TYPE_SEARCH,
+        VIA_UNIQUE_ID_ON_SCOPE_PROGRAM_SEARCH,
+    } = searchPageActionTypes;
+
+    return {
+        searchViaUniqueIdOnScopeTrackedEntityType: ({ trackedEntityTypeId, formId }) => {
+            dispatch(
+                actionCreator(VIA_UNIQUE_ID_ON_SCOPE_TRACKED_ENTITY_TYPE_SEARCH)({ trackedEntityTypeId, formId }),
```

Comment:
Yeah I dont have a strong opinion on this either. It felt when I was writing this that I could have it in one file since they are not being reused. Also sometimes it feels that I end up having many tabs open and all belong to the same component. Like the container file the component file the types and the actions. So I thought maybe I could try it out here and see how you also feel about this. 

However later on I am refactoring this. This will be on the ninth PR of this chain. Is this a blocker for you?